Resources/Prototypes/_RMC14/Entities/Objects/Devices/command_tablet.yml

```
@@ -0,0 +1,18 @@
+- type: entity
+  parent: BaseItem
+  id: CMCotablet
```

```suggestion
  id: RMCTabletCO
```

please, use `RMC` instead of `CM`.